UI/Web/src/assets/langs/en.json

```diff
@@ -54,6 +54,8 @@
         "default-age-restriction-label": "Age rating",
         "default-age-restriction-tooltip": "Maximum age rating shown to new users",
         "no-restriction": "{{restriction-selector.no-restriction}}",
+        "default-roles-label": "Roles",
+        "default-libraries-label": "Libraries",
```

Comment:
Avoid duplicating strings, you have "Libraries" in multiple places. This forces localization to happen multiple times. Re-use existing translations via {{key.key}}